citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.jsp

```diff
@@ -185,81 +194,115 @@ ${author.lastName}<c:if test="${not empty author.firstName}">, ${author.firstNam
+<td width="20%">Series Title</td>
 <td><form:input path="seriesTitle" type="text" class="form-control" placeholder="Series Title" value="${not empty form.seriesTitle ? form.seriesTitle : citation.seriesTitle}" /></td>
 </tr>
+</c:if>
+<c:if test="${fn:contains(fields, 'seriesText') }">
 <tr>
-<td>Series Text</td>
+<td width="20%">Series Text</td>
 <td><form:input path="seriesText" type="text" class="form-control" placeholder="Series Text" value="${not empty form.seriesText ? form.seriesText : citation.seriesText}" /></td>
 </tr>
+</c:if>
+<c:if test="${fn:contains(fields, 'url') }">
 <tr>
-<td>URL</td>
+<td width="20%">URL</td>
 <td><form:input path="url" type="text" class="form-control" placeholder="Url" value="${not empty form.url ? form.url : citation.url}" /></td>
 </tr>
+</c:if>
+<c:if test="${fn:contains(fields, 'language') }">
 <tr>
-<td>Language</td>
+<td width="20%">Language</td>
 <td><form:input path="language" type="text" class="form-control" placeholder="Language" value="${not empty form.language ? form.language : citation.language}" /></td>
 </tr>
+</c:if>
+<c:if test="${fn:contains(fields, 'doi') }">
 <tr>
-<td>DOI</td>
+<td width="20%">DOI</td>
 <td><form:input path="doi" type="text" class="form-control" placeholder="DOI" value="${not empty form.doi ? form.doi : citation.doi}" /></td>
 </tr>
+</c:if>
+<c:if test="${fn:contains(fields, 'issn') }">
 <tr>
-<td>ISSN</td>
+<td width="20%">ISSN</td>
 <td><form:input path="issn" type="text" class="form-control" placeholder="ISSN" value="${not empty form.issn ? form.issn : citation.issn}" /></td>
 </tr>
+</c:if>
+<c:if test="${fn:contains(fields, 'archive') }">
 <tr>
-<td>Archive</td>
+<td width="20%">Archive</td>
 <td><form:input path="archive" type="text" class="form-control" placeholder="Archive" value="${not empty form.archive ? form.archive : citation.archive}" /></td>
 </tr>
+</c:if>
+<c:if test="${fn:contains(fields, 'archiveLocation') }">
 <tr>
-<td>Archive Location</td>
+<td width="20%">Archive Location</td>
 <td><form:input path="archiveLocation" type="text" class="form-control" placeholder="Archive Location" value="${not empty form.archiveLocation ? form.archiveLocation : citation.archiveLocation}" /></td>
 </tr>
+</c:if>
+<c:if test="${fn:contains(fields, 'libraryCatalog') }">
 <tr>
-<td>Library Catalog</td>
+<td width="20%">Library Catalog</td>
 <td><form:input path="libraryCatalog" type="text" class="form-control" placeholder="Library Catalog" value="${not empty form.libraryCatalog ? form.libraryCatalog : citation.libraryCatalog}" /></td>
 </tr>
+</c:if>
+<c:if test="${fn:contains(fields, 'callNumber') }">
 <tr>
-<td>Call Number</td>
+<td width="20%">Call Number</td>
 <td><form:input path="callNumber" type="text" class="form-control" placeholder="Call Number" value="${not empty form.callNumber ? form.callNumber : citation.callNumber}" /></td>
 </tr>
+</c:if>
+<c:if test="${fn:contains(fields, 'rights') }">
 <tr>
-<td>Rights</td>
+<td width="20%">Rights</td>
```

Comment:
should be enough to give only one td 20% (one that is always shown), or if we want to change the width, we have to change every single td.